src/tools/Tools.mjs

```diff
@@ -75,7 +79,11 @@ export default class Tools {
         }
 
         let factory = (cb, stage) => {
-            cb(null, this.createShadowRect(stage, w, h, radius, blur, margin));
+            if (Utils.isWeb) {
```

Comment:
I don't see createShadowRect in platform namespace anymore.

Reply:
stage.platform.createShadowRect is now in the Lightning-Spark repo platform area.